games-fps/freedm-data/freedm-data-0.12.1-r1.ebuild

```diff
@@ -0,0 +1,56 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{8,9,10} )
+
+inherit prefix python-any-r1 xdg
+
+DESCRIPTION="Game resources for FreeDM"
+HOMEPAGE="https://freedoom.github.io"
+SRC_URI="https://github.com/freedoom/freedoom/archive/v${PV}.tar.gz -> freedoom-${PV}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~x86"
+
+BDEPEND="
+	$(python_gen_any_dep 'dev-python/pillow[${PYTHON_USEDEP},zlib]')
+	app-text/asciidoc
+	games-util/deutex[png]"
+
+S="${WORKDIR}/freedoom-${PV}"
+
+DOOMWADPATH=share/doom
+
+python_check_deps() {
+	has_version -b "dev-python/pillow[${PYTHON_USEDEP},zlib]"
+}
+
+src_prepare() {
+	xdg_src_prepare
+
+	hprefixify dist/freedoom
+}
+
+src_compile() {
+	emake wads/freedm.wad \
+		freedm.6 \
+		{NEWS,README}.html
+}
+
+src_install() {
+	emake install-freedm \
+		prefix="${ED}/usr/" \
+		bindir="bin/" \
+		docdir="share/doc/${P}" \
```

Comment:
Normally should use `${PF}` rather than `${P}` for doc dirs.

With the new revision, leads to:
```
 * QA Notice: The ebuild is installing to one or more unexpected paths:
 * 
 *   /usr/share/doc/freedm-data-0.12.1
```